internal/database/database-shelf.go

```diff
@@ -353,6 +355,67 @@ func (db *DB) ShelfListRowsPaginated(page int, rows int) ([]*items.ListRow, erro
 	return shelfRows, nil
 }
 
+func (db *DB) ShelfSearchListRowsPaginated(page int, rows int, search string) (shelfRows []*items.ListRow, found int, err error) {
+	// shelfRows = make([]*items.ListRow, 0)
+	// found = 0
+	// err = nil
+	if page < 1 {
```

Comment:
this belong to the shelf handlder and should not be placed hier (i should add it)